dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/controller/TempTestController.java

```diff
@@ -111,6 +131,59 @@ public String forwardConcorContribution(@RequestBody ConcorContributionReqForDrc
         return "OK";
     }
 
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @PostMapping(value = "/send-contributions")
+    @Operation(description = "<h3> ** Use with caution, only if you are certain !!! ** </h3>"
+        + "*This operation is not part of normal processing*<br>"
+        + "*It bypasses normal processing and can send production data to a third party*<br>"
+        + "<ul>"
+        + "<li>Obtains XML for each concor contribution ID in the list.</li>"
+        + "<li>Sends it to the DRC</li>"
+        + "<li>Does NOT create a Contributions File for the sent concor contributions.</li>"
+        + "<li>Does NOT update each successfully processed concor contribution to SENT in MAAT.</li>"
+        + "</ul>"
+        + "*Only to be used during testing or fault-finding*<br><br>"
+        + "When given a list of Concor Contribution IDs, get the XML for each and send them to the DRC"
+        + "Returns the list of Concor Contribution Entries (ID + XML) that were sent to DRC")
```

Comment:
I like this description. I think it would be good to have something similar warnings on the `forwardFdc` and `forwardConcorContribution` methods. What do you think?

Reply:
Done.